src/app/components/list/view-switcher/index.html

```diff
@@ -106,7 +106,30 @@ <h3>Components</h3>
     >
     </sky-demo-page-module-info>
 
-  </sky-tab>
+    <sky-demo-page-properties
```

Comment:
@Blackbaud-AlexKingman, do we need to create any additional development reference docs for this component? These were the only public properties I saw when I scanned the code, but let me know if you were planning on any additional info here or if we need to add anything about the component that contains this one.

Reply:
The properties are correct but we probably should call out somewhere what this is for. You use this when using the view switcher with a custom view that does not have a built in configuration to show up automatically. (Currently only the grid shows up automatically. The plan was for repeater and cards to do the same but not sure about that now that we may be doing something different there). 

I think this goes back to my other comment regarding us figuring out what we want to document for custom views.

Reply:
I fleshed out the description of the `view` property to try to call out the point you made. Do you think this works?

Reply:
LGTM